test/suites/integration/bsp/multi-volunteer/multi-volunteer.test-1.ts

```diff
@@ -0,0 +1,90 @@
+import type { SubmittableExtrinsic } from "@polkadot/api/types";
+import type { ISubmittableResult } from "@polkadot/types/types";
+import { strictEqual } from "node:assert";
+import invariant from "tiny-invariant";
+import {
+  alice,
+  bob,
+  bspThreeKey,
+  bspThreeSeed,
+  bspTwoKey,
+  bspTwoSeed,
+  charlie,
+  describeBspNet,
+  type EnrichedBspApi,
+  ShConsts,
+  sleep
+} from "../../../../util";
+
+describeBspNet("BSPNet: Mulitple BSP Volunteering - 3", ({ before, it, createUserApi }) => {
+  let api: EnrichedBspApi;
+
+  before(async () => {
+    api = await createUserApi();
+  });
+
+  // Test below seems to be failing. sh-bsp isn't volunteering to requests even though logs claim to
```

Comment:
Is this still the case? From your assertions it seems that it is working.

Reply:
whoops, bad copypasta here